addons/openebs/template/testgrid/k8s-docker-localpv.yaml

```
@@ -104,10 +104,10 @@
       version: "1.12.0"
   upgradeSpec:
     kubernetes:
-      version: "1.24.x"
+      version: "1.21.x"
     weave:
       version: "latest"
-    docker:
+    containerd:
```

```suggestion
    docker:
```

because otherwise we're also testing docker->containerd here

---

yeah, makes total sense.